src/StackExchange.Redis/ClientInfo.cs

```diff
@@ -180,6 +180,21 @@ public ClientType ClientType
             }
         }
 
+        /// <summary>
+        /// Client RESP protocol version. Added in Redis 7.0
+        /// </summary>
+        public string? ProtocolVersion { get; private set; }
+
+        /// <summary>
+        /// Client library version. Added in Redis 8.0
+        /// </summary>
+        public string? LibraryName { get; private set; }
+
+        /// <summary>
+        /// Client library version. Added in Redis 8.0
```

Comment:
I believe the server will support the new SETINFO subcommand in Redis 7.2 (not 8.0)

Reply:
+1 - fixed